include/gz/math/Inertial.hh

```diff
@@ -123,6 +166,52 @@ namespace gz
         return R * this->massMatrix.Moi() * R.Transposed();
       }
 
+      /// \brief Spatial mass matrix for body B. It does not include fluid
```

Comment:
this matrix should be expressed in frame F to match the `Moi()` API just above, and the documentation should explicitly mention that

Reply:
deac75cebb3a89576101407e79bb9aac853766b7